src/widget/core.rs

```diff
@@ -498,6 +498,11 @@ impl Pod {
         }
     }
 
+    /// Get the widgets size (as returned by the layout method)
+    pub fn get_size(&mut self) -> Size {
```

Comment:
I still think we should omit `get_` prefixes to stay consistent :)
```suggestion
    pub fn size(&mut self) -> Size {
```

Reply:
I've updated this